### DISCUSSION TITLE
v1.5.0-preview1

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#4194] [Announcements]
## 1.5 Preview 1 (1.5.240205001-preview1) 🔃

The first preview of WinAppSDK 1.5 is now available! Check out the [release notes](https://aka.ms/windowsappsdk/1.5/1.5-preview1/release-notes) for more information on the new & updated features and known issues.

### Updating to this version
The 1.5-preview1 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.5.240205001-preview1

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.5.0-preview1'>v1.5.0-preview1</a>.</em>

**Comment by Marv51** [comment]
@duncanmacmichael 

I think you have a typo in the first line? 
"1.5 Preview **2** (1.5.240205001-preview1) 🔃" it should be Preview **1**

**Reply by duncanmacmichael** [reply]
Ha, yes that was a typo, thanks for catching it :)

**Comment by christosk92** [comment]
Thanks ! The SDK is getting better and better each update. 
Makes me confident in the future of the WinAppSdk. I Hope the team can continue the hard work they're putting in.

**Reply by duncanmacmichael** [reply]
Thanks for the kind words! :)
